study.md

```diff
@@ -50,8 +50,38 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
-```
+Data represented with JSON looks like an object, but it is only a
+string.
+myKitchenObj = {
+    "kitchen": {
+        "cupboard": {
+          "bowl1": "bowl",
+          "bowl2": "bowl",
+          "bowl3": "bowl",
+          "plate1": "plate",
+          "plate2": "plate",
+          "plate3": "plate"
+        },
+		"electricStove": {
+      "heatingElement1": "heatingElement1",
+      "heatingElement2": "heatingElement2",
+      "heatingElement3": "heatingElement3",
+      "heatingElement4": "heatingElement4",
+    },
+    "herbsAndSpices": {
```

Comment:
again here the herbs and spices could be two arrays, one for herbs and one for spices.